CondTools/CTPPS/test/ppsTimingCalibrationWriter_cfg.py

```diff
@@ -0,0 +1,39 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('test')
+
+process.source = cms.Source('EmptyIOVSource',
+    timetype = cms.string('runnumber'),
+    firstValue = cms.uint64(1),
+    lastValue = cms.uint64(1),
+    interval = cms.uint64(1)
+)
+
+# load calibrations from JSON file
+process.load('CondFormats.CTPPSReadoutObjects.ppsTimingCalibrationESSource_cfi')
+process.ppsTimingCalibrationESSource.calibrationFile = cms.FileInPath('RecoCTPPS/TotemRPLocal/data/timing_offsets_ufsd_2018.dec18.cal.json')
+
+# output service for database
+process.load('CondCore.CondDB.CondDB_cfi')
+process.CondDB.connect = 'sqlite_file:totemTiming_calibration.sqlite' # SQLite output
+#process.CondDB.connect = 'oracle://cmsprep/TimingCalibration' # Oracle output
+#process.CondDB.connect = 'frontier://cmsfrontier.cern.ch:8000/FrontierPrep/TimingCalibration' # Frontier output
+
+process.PoolDBOutputService = cms.Service('PoolDBOutputService',
+    process.CondDB,
+    timetype = cms.untracked.string('runnumber'),
+    toPut = cms.VPSet(
+        cms.PSet(
+            record = cms.string('PPSTimingCalibrationRcd'),
+            tag = cms.string('TotemTimingCalibration'),
+            label = cms.string('UFSD')
```

Comment:
I don't think it is possible to add label attribute to tag in local sqlite file. Did you check whether the label worked?

Reply:
Hi @tocheng !
Indeed, no reference to this label in any attribute (even a `strings output_file.sqlite | grep -i UFSD` did not mention anything...). Thanks!
This `label` will be removed for the next iteration.